Wikipedia/Code/WMFFindInPageKeyboardBar.m

```diff
@@ -16,6 +16,18 @@ @interface WMFFindInPageKeyboardBar() <UITextFieldDelegate>
 
 @implementation WMFFindInPageKeyboardBar
 
+- (void)awakeFromNib {
+    [super awakeFromNib];
+    [self hideCopyPasteIcons];
+}
+
+- (void)hideCopyPasteIcons {
```

Comment:
This may not be what u think - it causes an extra 3 icons to appear in a particularly poor place on iPad.